Assets/Styles/abstracts/_mixins.scss

```diff
@@ -0,0 +1,256 @@
+/// @param {Map} $breakpoint-values - The dictionary where the breakpoints are keys and the values will be applied.
+/// @param {Bool} $horizontal [true] - Defines whether horizontal values should be applied or not.
+/// @param {Bool} $vertical [false] - Defines whether vertical values should be applied or not.
+/// @param {Bool} $negative [false] - Defines whether the values should be applied will be negated or not
+///                                   based on the assumption that the dictionary holds positive values only.
+/// @require $property, $breakpoint-values
+@mixin breakpoint-side-property($property, $breakpoint-values, $horizontal: true, $vertical: false, $negative: false) {
+    @each $breakpoint, $values in $breakpoint-values {
+        $value: if($negative, -$values, $values);
+
+        @include media-breakpoint-up($breakpoint) {
+            @if $horizontal == true {
+                #{$property}: {
+                    left: $value;
+                    right: $value;
+                }
+            }
+
+            @if $vertical == true {
+                #{$property}: {
+                    top: $value;
+                    bottom: $value;
+                }
+            }
+        }
+    }
+}
+
+/// Creates a horizontal or vertical (or both) margin based on a breakpoint-value dictionary
+/// with an additional option to apply negative values.
+/// @param {Map} $breakpoint-values - The dictionary where the breakpoints are keys and the values will be applied.
+/// @param {Bool} $horizontal [true] - Defines whether horizontal values should be applied or not.
+/// @param {Bool} $vertical [false] - Defines whether vertical values should be applied or not.
+/// @param {Bool} $negative [false] - Defines whether the values should be applied will be negated or not
+///                                   based on the assumption that the dictionary holds positive values only.
+/// @require $breakpoint-values
+@mixin breakpoint-margin($breakpoint-values, $horizontal: true, $vertical: false, $negative: false) {
+    @include breakpoint-side-property(
+        $property: "margin",
+        $breakpoint-values: $breakpoint-values,
+        $horizontal: $horizontal,
+        $vertical: $vertical,
+        $negative: $negative);
+}
+
+/// Creates a horizontal or vertical (or both) padding based on a breakpoint-value dictionary
+/// with an additional option to apply negative values.
+/// @param {Map} $breakpoint-values - The dictionary where the breakpoints are keys and the values will be applied.
+/// @param {Bool} $horizontal [true] - Defines whether horizontal values should be applied or not.
+/// @param {Bool} $vertical [false] - Defines whether vertical values should be applied or not.
+/// @param {Bool} $negative [false] - Defines whether the values should be applied will be negated or not
+///                                   based on the assumption that the dictionary holds positive values only.
```

Comment:
With all the great work that's been done here, it may be nitpicking, but only in an effort for consistency: The comment blocks should all be wrapped to 120 characters which the [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) VS plugin.

Oh, I'm talking to a Rider user! What can we do about this?

Reply:
There is a [Wrap to Column](https://plugins.jetbrains.com/plugin/7234-wrap-to-column/) plugin, but it doesn't indent these Doxygen style comments as they should be (like in line 114 here). I doubt Rewrap does that either? So here it has to be done manually.

Reply:
Rewrap wraps them to fit 120 chars, but does not keep the nice indentation like for the `$negative` parameter. I can fix that once I'll be working on the project for real ;-)